src/main/java/org/opensearch/knn/common/KNNConstants.java

```diff
@@ -24,6 +24,7 @@ public class KNNConstants {
     public static final String KNN = "knn";
     public static final String VECTOR = "vector";
     public static final String K = "k";
+    public static final String EF_SEARCH = "ef_Search";
```

Comment:
```suggestion
    public static final String EF_SEARCH = "ef_search";
```

Reply:
We also have another constance METHOD_PARAMETER_EF_SEARCH , can we not re-factor and use it in both places?

Reply:
I missed the existing constant, will refactor it for consistency